msm/src/proof.rs

```diff
@@ -105,14 +107,20 @@ impl<G: KimchiCurve> Witness<G> {
                     })
                     .collect::<Vec<_>>(),
             },
+            mvlookups: vec![LookupWitness::<G::ScalarField>::random(domain)],
         }
     }
 }
 
 #[derive(Debug, Clone)]
 pub struct Proof<G: KimchiCurve, OpeningProof: OpenProof<G>> {
+    // Columns/PlonK argument
     pub(crate) commitments: WitnessColumns<PolyComm<G>>,
     pub(crate) zeta_evaluations: WitnessColumns<G::ScalarField>,
     pub(crate) zeta_omega_evaluations: WitnessColumns<G::ScalarField>,
```

Comment:
I honestly like this so much more than `kimchi`'s `evaluations` which contains `WitnessColumns` of a pair. Slightly verbose stuff is easily understandable, and we're not even close to the verbosity level of Java so I wouldn't worry :)